src/google_cloud_sdk_emulators/mod.rs

```diff
@@ -1,19 +1,21 @@
 use testcontainers::{core::WaitFor, Image, ImageArgs};
 
-const NAME: &str = "google/cloud-sdk";
-const TAG: &str = "362.0.0-emulators";
+const NAME: &str = "gcr.io/google.com/cloudsdktool/google-cloud-cli";
+const TAG: &str = "471.0.0-emulators";
```

Comment:
- Do we really need to bump the version?
- Are we sure it doesn't break anything for users of the module?

Tag can be customized on user end and usually we don't bump versions if it's not necessary. Because it may break something for users using default instance.

Reply:
> Do we really need to bump the version?

No, we don't.

> Are we sure it doesn't break anything for users of the module?

No because there aren't a lot of tests, but this is relatively unlikely (at least from my experience). It should actually provide some benefits since it will upgrade some of the emulators which bring additional functionality.

I don't mind leaving this as-is, no strong opinion here either.

Reply:
I've reverted the bump in https://github.com/testcontainers/testcontainers-rs-modules-community/pull/117/commits/67590db5c0beb7216bebe0f202a73e61f297be00